src/app/helptext/apps/apps.ts

```diff
@@ -555,6 +555,24 @@ export default {
     },
     pulling: T('Pulling...'),
   },
+  thirdPartyRepoWarning: {
+    btnMsg: T('Continue'),
+    cancelMsg: T('Cancel'),
+    title: T('Warning'),
+    message: T(`iXsystems does not audit or otherwise validate the contents of third-party applications catalogs. \
+    It is incumbent on the user to verify that the new catalog is from a trusted source and that the third-party \
+    properly audits its chart contents. Failure to exercise due diligence can expose the user and their data to \
+    the some or all of the following:<br/>\
+    <ul>
+      <li>Malicious software</li>
+      <li>Broken services on TrueNAS host</li>
+      <li>Service disruption on TrueNAS host</li>
+      <li>Broken filesystem permissions on Host or within application</li>
```

Comment:
It might be best to remove this one.
As it gives the impression that iX-Systems does guarantee this, which is factually not the case (or not completely at the very least).

(To give insight: Bitnami and TrueCharts, for example, at least ship tools to have us manage the permissions of user-added storage, where iX does not)

Reply:
@Ornias1993 Not sure I understand. What would make user think that ixsystems guarantees broken  filesystem permissions on Host or within application?

Reply:
This statement is staying. Apps can have access to the filesystem and do whatever they want with it.

Reply:
I should've indeed been more clear @william-gr sorry.

What i meant was the last part:
`on Host or within application`

Primarily the "within applications" part.

I would suggest the following:
```suggestion
      <li>Broken filesystem permissions</li>
```